src/apiCall.js

```diff
@@ -8,21 +8,25 @@ const SuperClient = require('./superClient')
 
 const processedExpectations = ['status', 'json', 'headers', 'error']
 
-function getItFunction(req) {
+function getItFunction(req, itApi = it) {
   const useOnly = req.only
   const useSkip = req.skip
-  let itMethod = it
+  let itMethod = itApi
   if (useOnly) {
-    itMethod = it.only
+    itMethod = itApi.only
   }
 
   if (useSkip) {
-    itMethod = it.skip
+    itMethod = itApi.skip
   }
   return itMethod
 }
 
-module.exports = function apiCall(file, apiPort) {
+function getItName(req) {
+  return req.name || req.call
+}
+
+module.exports = function apiCall(file, apiPort, itApi = it) {
```

Comment:
I added this `itApi` so it is more natural to mock/fake is in tests; by default, it will use the global `it`.